client.go

```diff
@@ -20,13 +20,13 @@ import (
 	"context"
 
 	gpb "github.com/GreptimeTeam/greptime-proto/go/greptime/v1"
-	"google.golang.org/grpc"
-
+	"github.com/GreptimeTeam/greptimedb-ingester-go/bulk"
```

Comment:
It seems the order is incorrect. Also, it may be better to make the check in the lint.